test/__snapshots__/fixtures.test.ts.snap

```diff
@@ -209,58 +209,177 @@ Array [
 exports[`fixtures should match all snapshots: 380.mdx 1`] = `
 Array [
   Object {
-    "column": 6,
-    "endColumn": 6,
-    "endLine": 74,
-    "line": 48,
-    "message": "Expected an assignment or function call and instead saw an expression.",
-    "messageId": "unusedExpression",
-    "nodeType": "ExpressionStatement",
-    "ruleId": "no-unused-expressions",
+    "column": 28,
+    "endColumn": 8,
+    "endLine": 70,
+    "fix": Object {
+      "range": Array [
+        896,
+        1393,
+      ],
+      "text": "
+        <aui-tab-group [(tab)]=\\"tab\\">
+          <aui-tab
+            name=\\"a\\"
+            label=\\"Tab 0\\"
+          >
+            <aui-card> Content 1</aui-card>
+          </aui-tab>
+          <aui-tab
+            name=\\"b\\"
+            label=\\"Tab 1\\"
+          >
+            <aui-card> Content 2</aui-card>
+          </aui-tab>
+          <aui-tab
+            name=\\"c\\"
+            label=\\"Tab 2\\"
+          >
+            <aui-card> Content 3</aui-card>
+          </aui-tab>
+        </aui-tab-group>
+      ",
+    },
+    "line": 49,
+    "message": "Templates should be properly indented.",
+    "messageId": "template-indent",
+    "nodeType": "TemplateLiteral",
+    "ruleId": "unicorn/template-indent",
```

Comment:
Another problem next to be resolved.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1815#issuecomment-1126881651